sourcegraph/about
# Fix title meta data in SG Overview PDF

## What does this PR do?

Closes #5186 

## How to test
1. View preview link
2. Click on the overview pdf link in the footer
3. Ensure that the tab title says "Sourcegraph Overview"
